FWCore/Framework/test/stubs/TestStreamProducers.cc

```diff
@@ -664,14 +664,20 @@ namespace edmtest {
       //Using mutable since we want to update the value.
       mutable std::atomic<unsigned int> m_value;
       mutable std::atomic<unsigned int> m_expectedValue;
+      // Set only in constructor, framework runs constructors serially
+      CMS_THREAD_SAFE mutable edm::EDPutTokenT<unsigned int> m_putToken;
```

Comment:
Interesting problem where stream module wants to do a global put. Hmm.

Reply:
I copied the pattern from
https://github.com/cms-sw/cmssw/blob/6d0e276c82e1a460a634b92f141252d534adc1c2/DQMServices/Core/interface/DQMEDAnalyzer.h#L28-L34